packages/squidex/schema/schemas/users.json

```diff
@@ -483,8 +495,11 @@
               "allowedValues": [
                 "Lead PI",
                 "Co-Investigator",
-                "Project Manager",
                 "Collaborator",
+                "Lead PI (Core Leadership)",
+                "Co-PI (Core Leadership)",
+                "Project Manager",
+                "Collaborating PI",
```

Comment:
Since we need to migrate the roles to the new designation I am keeping the previous ones on this PR.